OOP_PHONE.cpp

```diff
@@ -0,0 +1,277 @@
+                    }
+                }
+                else if (sign == "Descending")
+                {
+                    if (elem == "Surname")
+                    {
+                        tmp = arr[j]->surname[0] < arr[j + 1]->surname[0];
+                    }
+                    else
+                    {
+                        tmp = arr[j]->name[0] < arr[j + 1]->name[0];
+                    }
+                }
+                else
+                {
+                    cout << "Error";
+                }
+                if (tmp)
+                {
+                    Person *x = arr[j + 1];
+                    arr[j + 1] = arr[j];
+                    arr[j] = x;
+                }
+            }
+        }
+        return arr;
+    }
+
+    Person **sortPhoneNumber(Person **arr, string sign, int count)
+    {
+        for (int i = 0; i < count; i++)
+        {
+            for (int j = 0; j < count - 1 - i; j++)
+            {
+                bool tmp;
+                if (sign == "Ascending")
+                {
+                    tmp = arr[j]->phoneNumber[0] > arr[j + 1]->phoneNumber[0] ||
+                          arr[j]->phoneNumber[1] > arr[j + 1]->phoneNumber[1] ||
+                          arr[j]->phoneNumber[2] > arr[j + 1]->phoneNumber[2];
+                }
+                else if (sign == "Descending")
+                {
+                    tmp = arr[j]->phoneNumber[0] < arr[j + 1]->phoneNumber[0] ||
+                          arr[j]->phoneNumber[1] < arr[j + 1]->phoneNumber[1] ||
+                          arr[j]->phoneNumber[2] < arr[j + 1]->phoneNumber[2];
+                }
+                if (tmp)
+                {
+                    Person *x = arr[j + 1];
+                    arr[j + 1] = arr[j];
+                    arr[j] = x;
+                }
+            }
+        }
+        return arr;
+    }
+
+    void sort(Person **arr, int count)
+    {
+        string x;
+        string y;
+        cout << "Please choose an ordering to sort: 'Ascending' or 'Descending'." << endl;
+        cin >> x;
+        if (x == "Ascending" || x == "Descending")
+        {
+            cout << "Please choose criteria: 'Name', 'Surname' or 'PhoneNumberCode'." << endl;
+            cin >> y;
+            cout << endl;
+            if (y == "Name" || y == "Surname")
+            {
+                sortName(arr, x, y, count);
+            }
+            else if (y == "PhoneNumberCode")
+            {
+                sortPhoneNumber(arr, x, count);
+            }
+            else
+            {
+                cout << endl
+                     << "Sorting failed" << endl;
+                cout << endl;
+            }
+        }
+        else
+        {
+            cout << endl
+                 << "Sorting failed" << endl;
+            cout << endl;
+        }
+    }
+};
+
+class File : public Helper
+{
+public:
+    int fileLine(string fileName)
```

Comment:
what does this method do? If it counts file lines then, please name it accordingly